subprojects/docs/src/docs/userguide/troubleshooting.adoc

```diff
@@ -15,29 +15,169 @@
 [[troubleshooting]]
 == Troubleshooting
 
+The following is a collection of common issues and suggestions for addressing them. You can get other tips and search the link:https://discuss.gradle.org/c/help-discuss[Gradle forums] and link:https://stackoverflow.com/questions/tagged/gradle[StackOverflow #gradle] answers, as well as Gradle documentation from link:https://help.gradle.org/[help.gradle.org].
 
-[NOTE]
-====
- 
-This chapter is currently a work in progress.
- 
-====
+=== Troubleshooting Gradle installation
 
-When using Gradle (or any software package), you can run into problems. You may not understand how to use a particular feature, or you may encounter a defect. Or, you may have a general question about Gradle.
+If you followed the <<installation,installation instructions>>, and aren’t able to execute your Gradle build, here are some tips that may help.
 
-This chapter gives some advice for troubleshooting problems and explains how to get help with your problems.
+If you installed Gradle outside of just invoking the <<gradle_wrapper,Gradle wrapper>>, you can check your Gradle installation by running `gradle --version` in a terminal.
```

Comment:
We are capitalizing the word Wrapper usually. See https://docs.gradle.org/nightly/userguide/gradle_wrapper.html.